specification/appconfiguration/data-plane/Microsoft.AppConfiguration/preview/2023-05-01-preview/appconfiguration.json

```diff
@@ -0,0 +1,2079 @@
+      }
+    }
+  },
+  "definitions": {
+    "Key": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The name of the key."
+        }
+      }
+    },
+    "KeyValue": {
+      "type": "object",
+      "properties": {
+        "key": {
+          "type": "string",
+          "description": "The key of the key-value."
+        },
+        "label": {
+          "type": "string",
+          "description": "The label the key-value belongs to."
+        },
+        "content_type": {
+          "type": "string",
+          "description": "The content type of the value stored within the key-value."
+        },
+        "value": {
+          "type": "string",
+          "description": "The value of the key-value."
+        },
+        "last_modified": {
+          "format": "date-time",
+          "type": "string",
+          "description": "A date representing the last time the key-value was modified."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The tags of the key-value"
+        },
+        "locked": {
+          "type": "boolean",
+          "description": "Indicates whether the key-value is locked."
+        },
+        "etag": {
+          "type": "string",
+          "description": "A value representing the current state of the resource."
+        }
+      }
+    },
+    "Snapshot": {
+      "type": "object",
+      "required": [
+        "filters"
+      ],
+      "properties": {
+        "name": {
+          "description": "The name of the snapshot.",
+          "type": "string",
+          "readOnly": true
+        },
+        "status": {
+          "description": "The current status of the snapshot.",
+          "type": "string",
+          "enum": [
+            "provisioning",
+            "ready",
+            "archived",
+            "failed"
+          ],
+          "x-nullable": false,
+          "x-ms-enum": {
+            "name": "SnapshotStatus",
+            "modelAsString": true
+          },
+          "readOnly": true
+        },
+        "filters": {
+          "description": "A list of filters used to filter the key-values included in the snapshot.",
+          "type": "array",
+          "minItems": 1,
+          "maxItems": 3,
+          "items": {
+            "$ref": "#/definitions/KeyValueFilter"
+          }
+        },
+        "composition_type": {
+          "description": "The composition type describes how the key-values within the snapshot are composed. The 'key' composition type ensures there are no two key-values containing the same key. The 'key_label' composition type ensures there are no two key-values containing the same key and label.",
+          "type": "string",
+          "enum": [
+            "key",
+            "key_label"
+          ],
+          "x-nullable": false,
```

Comment:
Another instance of `x-nullable: false` that should be removed. I see there are also more so please do this throughout.